### ISSUE TITLE
MacOS Hypervisor Framework broken with Qemu.

### ISSUE THREAD

**Issue by ExternalReality** [nanovms/ops#268]
On mac, if acceleration is enabled in the run-time config, Ops will add the `-accel hvf -cpu host` flags to its generated Qemu Command. This causes an error.

```
bad size

signal: abort trap
```

Given this once worked unfailingly, I am guessing its a kernel regression.

```sh
ops run -v hello -c config.json
Downloading.. /Users/externalreality/.ops/0.1.10/nanos-release-darwin-0.1.10.tar.gz
 506.10 KiB / 506.10 KiB [====================================================================================] 100.00% 1.20 MiB/s 0s
Finding dependent shared libs
booting /Users/externalreality/.ops/images/hello.img ...
qemu-system-x86_64 -drive file=/Users/externalreality/.ops/images/hello.img,format=raw,if=none,id=hd0 -device virtio-blk,drive=hd0 -device virtio-net,netdev=n0 -netdev user,id=n0 -accel hvf -cpu host -nodefaults -no-reboot -device isa-debug-exit -m 2G -display none -serial stdio
bad size

signal: abort trap
```

**Comment by mkhon** [comment]
This one is a bug in qemu hvf

**Comment by mkhon** [comment]
Fixed in nanovms/qemu#1

**Comment by wjhun** [comment]
@ExternalReality Which version of qemu are you using here? And which were you using before the regression?

**Comment by mkhon** [comment]
Fixed in nanovms/nanos#702
